js/adapt-languagePicker.js

```diff
@@ -61,7 +61,7 @@ define([
     
     function setupNavigationView () {
         var courseGlobals = Adapt.course.get('_globals')._extensions;
-        var navigationBarLabel = 'Select course language';
+        var navigationBarLabel = '';
         if (_.has(courseGlobals, '_languagePicker')) {
```

Comment:
Just as a small note.  
```javascript
if (_.has(courseGlobals, '_languagePicker')) {
``` 
can be written as
```javascript
if (courseGlobals._languagePicker) {
```
which avoids a whole lot of extra computation for making the ``"_languagePicker"`` string, looking up the ``has`` property on the ``_`` object, executing the ``has`` function, making the function closure, do all the extra comparisons and return:

```javascript
_.has = function(obj, key) {
    return obj != null && hasOwnProperty.call(obj, key);
};
```